src/abi/TokenFactory.json

```diff
@@ -0,0 +1,5858 @@
+{
```

Comment:
Should this file be committed to git?

Reply:
Yea, we have moved to having the connect be knowledgable about the ABI of the methods and events it expects to interact with, rather than relying on ethconnect to translate them into friendly REST endpoints. All of the calls into ethconnect now use the raw API that requires full ABIs for the methods and events you want to interact with.

We've therefore checked in the full ABI as JSON, so individual pieces can be imported and extracted by the service. In theory we could copy-paste just a subset of the methods used into a JSON or JavaScript file, but it didn't feel totally necessary to do that extra transformation.